pkg/ebpf/events_pipeline.go

```diff
@@ -233,12 +241,7 @@ func (t *Tracee) decodeEvents(outerCtx context.Context, sourceChan chan []byte)
 				HostName:            string(bytes.TrimRight(ctx.UtsName[:], "\x00")),
 				CgroupID:            uint(ctx.CgroupID),
 				ContainerID:         containerInfo.ContainerId,
```

Comment:
Shouldn't we put ContainerId under ContainerData?

Reply:
It definitely could fit.
But we probably have a lot of users who would need to migrate the field, and it's a good indication for wether the fields inside `ContainerData` will be set.
`ContainerData` is only filled up with enrichment so it's easier to switch around (less existing consumers).

Reply:
If you are already changing this, better to do it once in accordance with #2870 and put it under event.context.container

Reply:
Yes.
Anyway we are going to break it, so better break it once

Reply:
Done